README.md

```diff
@@ -19,9 +19,13 @@ If you installed correctly, move your mouse to the left edge of the X-Plane wind
 * Automatic recording start or stop is disabled for about 5 minutes after you manually start or stop recording (see below).
 
 ### Manually starting and stopping recording
-* You can manually start and stop recording by clicking on the CAWR UI window.
+* You can manually start and stop recording by clicking on the recording circle in the CAWR UI window.
 * The litte circle turns red when recording, and the counter starts incrementing to show you how long the recording is.
-* Tap the CAWR UI again to stop recording.
+* Tap the recording circle again to stop recording.
+
+### Settings
+* Access the Settings screen by tapping on the letters CAWR at the top of the pop up UI.
```

Comment:
Mysterious UI :)

I would suggest adding an explicit "Settings" button.

Reply:
There's not much screen real estate for it though. Maybe a little gear graphic or something?